examples/live/generate_injections.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+
+import os, sys
+from pycbc.io import record
+import h5py
+from pycbc.inject import InjectionSet
+
+if os.path.exists('./test_inj.hdf'):
+    raise OSError("output-file already exists")
+
+# 2 injections 
+static_params = { 'f_lower': 18.0, 'f_ref': 20.0, \
+               'taper': 'start', 'ra': 45.0, 'dec': 45.0, 'inclination': 0.0, \
+               'coa_phase': 0.0, 'polarization': 0.0}
+
+samples = record.FieldArray(2, dtype=[('approximant', h5py.string_dtype(encoding='utf-8')), \
+                                      ('mass1', float), ('mass2', float), \
+                                      ('spin1z', float), ('spin2z', float), \
+                                      ('tc', float), ('distance', float)])
+
+samples['approximant'] = [u'SEOBNRv4', u'SpinTaylorT4']
```

Comment:
Since the following lines contain a bunch of unexplained "magic values", in cases like this you should add a comment saying where they come from. In this case, they are chosen to match two templates in the bank.